react/I18n/format.jsx

```diff
@@ -36,3 +37,6 @@ export const initFormat = (userLang, defaultLang = DEFAULT_LANG) => (
 
 export const formatLocallyDistanceToNow = date =>
   formatDistanceToNow(date, { locale: locales[lang] })
+
+export const formatLocallyDistanceToNowStrict = date =>
```

Comment:
The tests and the values are mostly copied from the ones for `formatLocallyDistanceToNow` above in the file so i didn't really choose the values, but I can work towards making them more human-readable, sure.

Also note that the link you gave points to a newer version of `date-fns` which apparently has a slightly different implementation. For example "should formatDistanceToNowStrict with medium value" returns 44 minutes in this PR, but it will return 45 minutes when we update `date-fns` (the value being about 44.52 minutes). So how the rounding is done is actually subject to change.

Reply:
I added some inline comments to make durations more human-readable.